chain/network/src/private_actix.rs

```diff
@@ -135,13 +155,27 @@ pub struct ValidateEdgeList {
     /// will add them.
     /// TODO(#5254): Simplify this process.
     pub(crate) sender: QueueSender<Edge>,
-    #[cfg(feature = "test_features")]
-    /// Feature to disable edge validation for purpose of testing.
-    pub(crate) adv_disable_edge_signature_verification: bool,
 }
 
-#[derive(actix::MessageResponse, Debug)]
-#[cfg_attr(feature = "test_features", derive(serde::Serialize))]
-pub struct GetRoutingTableResult {
-    pub edges_info: Vec<SimpleEdge>,
+impl PeerToManagerMsgResp {
+    pub fn as_routed_message_from(self) -> bool {
```

Comment:
```suggestion
    pub fn unwrap_routed_message_from(self) -> bool {
```

and same for other methods, `unwrap` is idiomatic name for panicky unwrapping an enum variant.